mlflow/pytorch/_pytorch_autolog.py

```diff
@@ -1,7 +1,7 @@
 from packaging.version import Version
+import os
```

Comment:
Can we revert this change?

Reply:
Good point! I'll revert back the one on the `mlflow.sklearn` module as well. :+1: